etc/kayobe/environments/ci-aio/kolla/globals.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Most development environments will use nested virtualisation, and we can't
+# guarantee that nested KVM support is available. Use QEMU as a lowest common
+# denominator.
+nova_compute_virt_type: qemu
+
+# Reduce the control plane's memory footprint by limiting the number of worker
+# processes to one per-service.
+openstack_service_workers: "1"
+openstack_service_rpc_workers: "1"
+
+docker_yum_baseurl: "{{ stackhpc_repo_docker_url }}"
+docker_yum_gpgkey: "https://download.docker.com/linux/{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %}/gpg"
```

Comment:
I suppose we're missing the container image tags here. I wonder if we can include the variables in etc/kayobe/kolla/globals.yml in here using e.g. `{{ lookup('template', kayobe_config_path ~ '/kolla/globals.yml' }}`.

Reply:
Actually, we merge this file in stackhpc/wallaby.